### PR TITLE
create .gitignore + test PR

### PR DESCRIPTION
**Pull request by ezgimez**
ignore double commits about .gitignore. just a mock PR overall

### CODE REVIEW

**Review by yp717 — COMMENTED** [review]
lgtm but do you want this merged in? or is this a duplicate PR?

**Review by Spruudis — APPROVED** [review]

**Comment by ezgimez** [comment]
not a duplicate PR.. I rebased the branch at some point (yes, I don't know git honestly) -- hence the duplication for .ignore.. hopefuly won't happen ever again

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# LaTeX temporary files
+*.aux
+*.log
+*.toc
+
+# PDF output - usually a bad idea to keep this in Git
+*.pdf
```
Comment (lhl2617) [inline .gitignore:7]: I usually gitignore only the output pdf -- figure pdfs will get ignored with this

**Comment by lhl2617** [comment]
see: https://github.com/github/gitignore/blob/master/TeX.gitignore

File: .gitignore (hunk 1 above)
Comment (ezgimez) [reply .gitignore:7]: figure pdfs will be inside each folder '00_xx'.. will this comment still gitignore them? if that's the case, we gotta change this line -- cuz I was not sure

Comment (lhl2617) [reply .gitignore:7]: yep, it will, this ignores all pdfs

Comment (ezgimez) [reply .gitignore:7]: fuck, how can we change it so that it ignores the only 'main' pdf then

**Comment by ezgimez** [comment]
hm, maybe we should have that .gitignore in the link you sent @lhl2617

File: .gitignore (hunk 1 above)
Comment (lhl2617) [reply .gitignore:7]: replace *.pdf with main.pdf

Comment (ezgimez) [reply .gitignore:7]: lol

**Pull request merged**